src/Component/Configuration/Loader/BabelLoader.php

```diff
@@ -28,6 +28,12 @@ public static function applyConfiguration(NodeBuilder $node_builder)
         $node_builder
             ->arrayNode('babel')
                 ->canBeDisabled()
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->arrayNode('presets')
```

Comment:
What I meant was, what happens to babel if nothing is specified? Does it automatically fallback to ES2015?

If babel doesn't do any transpiling out-of-the-box, please provide a default value here to prevent making backwards incompatible changes.